docs/modules/ROOT/pages/tmail-backend/jmap-extensions/publicAssets.adoc

```diff
@@ -31,8 +31,6 @@ It has the following fields:
 - *publicURI*: `String`. Server-set. The public URI serving the PublicAsset.
 - *size*: `UnsignedInt`. Server-set. The size of the PublicAsset, from the previously uploaded blob it refers to.
 - *contentType*: `String`. Server-set. The content type of the PublicAsset, from the previously uploaded blob it refers to.
-- *blobId*: `Id`. The blobId of the publicAsset in the object store.
-- *identityIds*: `Id[]|null`. Default to null. The identity ids that are using the PublicAsset.
```

Comment:
It's not because we don't return those fields in /set create method that they are not part of the object? Put that back please